fluentd/configs.d/input-ocp-audit.conf

```diff
@@ -0,0 +1,44 @@
+<source>
```

Comment:
We;ll have to figure out how to make this optional as  i expect this will add considerable load that we will want to 'opt-in'